setup.py

```diff
@@ -25,7 +25,7 @@
     package_data={
         "q2_amr": [
             "citations.bib",
-            "tests/data/*",
+            "card/tests/data/*",
```

Comment:
This is already covered by the line you changed below so you can remove it from here.